rocketpool/node/manage-fee-recipient.go

```diff
@@ -120,12 +121,15 @@ func (m *manageFeeRecipient) run(state *state.NetworkState) error {
 		m.log.Println("***ERROR***")
 		m.log.Printlnf("Error updating fee recipient files: %s", err.Error())
 		m.log.Println("Shutting down the validator client for safety to prevent you from being penalized...")
+		alerting.AlertFeeRecipientChanged(m.cfg, correctFeeRecipient, false)
 
 		err = validator.StopValidator(m.cfg, m.bc, &m.log, m.d)
 		if err != nil {
 			return fmt.Errorf("error stopping validator client: %w", err)
 		}
 		return nil
+	} else {
+		alerting.AlertFeeRecipientChanged(m.cfg, correctFeeRecipient, true)
```

Comment:
Same deal here- do the alerting between UpdateFeeRecipientFile and the conditional